install_config/aggregate_logging.adoc

```diff
@@ -468,81 +468,84 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-If you need to scale up the number of Elasticsearch instances your cluster uses
-it is not as simple as changing the number of replicas. This is due to the nature
-of Persistent volumes and how Elasticsearch is configured to store its data and
-recover the cluster.
+If you need to scale up the number of Elasticsearch instances your cluster uses,
+it is not as simple as changing the number of replicas. This is due to the
+nature of persistent volumes and how Elasticsearch is configured to store its
+data and recover the cluster.
 
-Instead, you will need to create a Deployment Configuration for each replica.
+Instead, you will need to create a deployment configuration for each replica.
 
-The Deployer created templates during installation with the Elasticsearch
-configurations provided to it: `logging-es-template` and `logging-es-ops-template`
-if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+The deployer created templates during installation with the Elasticsearch
+configurations provided to it: *logging-es-template* and
+*logging-es-ops-template* if the deployer was run with
+`*ENABLE_OPS_CLUSTER=true*`.
```

Comment:
@bfallonf This is more to let users know that behind the scenes these are the two templates the deployer uses to create the two separate clusters and that they'll need to use them as well.  There may be a better way of structuring this in the case where a user is directly linked to the scale up section.